nbuild/01.nbuild.md

```diff
@@ -0,0 +1,20 @@
+# 1 ``nbuild``
+## 1.1 What `nbuild` is for
+[//]: # (TODO: Add link to the getting-started for Nest)
+`nbuild` is an automated package builder.
+From a build manifest and source code, `nbuild` will abstract the build system used (cmake, autoconf, etc.), compile the software, get the executable files and files to install (binaries, libraries, manual pages, etc.) and wrap them in a package that Nest will know how to install: a `.nest` archive, called an npf file.
+It contains a `data.tar.gz` with all the files to install, if there are any, and a build manifest `manifest.toml`, containing basic information and metadata for this software (its name, version, description, licenses, etc.).
+
+All of this is done thanks to a powerful library included in `nbuild`: the standard compilation library, for which you can find the documentation [here](https://docs.raven-os.org/nbuild/master/stdlib.html).
+
+`nbuild` requires python3.6+ to run, and you will first need to install `nbuild`'s dependencies by running `pip install -r requirements.txt`.
```

Comment:
The recommend way to use nbuild is using it's docker image. It provides a working Raven-OS environment on any Linux distribution while still protecting the user's system from any failed compilation.

`docker run -it ravenos/nbuild`.